src/perform_step/kencarp_kvaerno_perform_step.jl

```diff
@@ -283,6 +289,7 @@ end
   if typeof(integrator.f) <: SplitFunction
     integrator.k[1] = integrator.fsalfirst
     integrator.fsallast = integrator.f(u, p, t+dt)
+    integrator.destats.nf += 1
```

Comment:
IMEX issue

Reply:
In KenCarp3, `f2` is calculated thrice and `f1` twice. So should I increment by 3?